docs/robotics/miscellaneous/create_urdf.md

```diff
@@ -0,0 +1,125 @@
+# Create URDF and PROTO for NAOv6
+
+For Webots we need a PROTO file which should contain 3D models and the scene graph of the NAOv6. The only source is a URDF from http://doc.aldebaran.com/2-8/family/nao_technical/kinematics_naov6.html#naov6-urdf-files. Meshes in OGRE mesh format can be found in e.g. the "C++ SDK" from https://developer.softbankrobotics.com/nao6/downloads/nao6-downloads-linux (in `share/alrobotmodel/meshes`). The URDF and meshes need to be converted to Webots PROTO.
+
+Download and build target `OgreXMLConverter` in https://github.com/OGRECave/ogre (e.g. `cmake --build build --target OgreXMLConverter`). `OgreXMLConverter` is able to convert OGRE mesh files into XML files containing the raw vertices and face vector indices. The resulting XML files can be converted into binary STL files with the following script:
+
+```py
+import collections
+import pathlib
+import struct
+import typing
+import xml.etree.ElementTree
```

Comment:
I think this file lives [here](https://github.com/HULKs/hulk/blob/main/webots/protos/meshes/xml_to_stl.py) already. Perhaps just replace it with a link?

Reply:
Linkified.